### PR TITLE
Copy paste fixing on lift documentation

### PR DESCRIPTION
**Pull request by tttp**

### CODE REVIEW

**Comment by tttp** [comment]
Not sure what license/cla is needed, but it's fixing a typo in the doc, do whatever you want with the change ;)

File: docs/lift/use-only-lift.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ When using PostgreSQL, you need to provide the following information to connect
 - **Password**: The password for the database user.
 - **SSL**: Whether or not your database server uses SSL.
 
-Once provided, the CLI will prompt you to select one of the existing **schemas** on your MySQL server for introspection.
+Once provided, the CLI will prompt you to select one of the existing **schemas** on your PostreSQL server for introspection.
```
Comment (janpio) [inline docs/lift/use-only-lift.md]:
```suggestion
Once provided, the CLI will prompt you to select one of the existing **schemas** on your PostgreSQL server for introspection.
```

**Comment by tttp** [comment]
the fix typo on the fix typo ;)
Thanks!

**Comment by nikolasburk** [comment]
Hey @tttp, thanks a lot for catching this and apologies for the delayed response! 🙏 we recently updated the docs and removed that particular docs page, so I'm closing this PR. Thanks again for your help, that's much appreciated!! 🙏

**Pull request closed without merging**